code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -13,6 +13,7 @@
  */
 var/global/list/datum/stack_recipe/metal_recipes = list ( \
 	new/datum/stack_recipe("stool", /obj/structure/chair/stool, one_per_turf = 1, on_floor = 1), \
+	new/datum/stack_recipe("bar stool", /obj/structure/chair/stool/bar, one_per_turf = 1, on_floor = 1), \
```

Comment:
if you are going to stay stuff for the sake of bothering me, just fuck off, please.